src/Cake.Core/ICakeEnvironment.cs

```diff
@@ -51,5 +52,11 @@ public interface ICakeEnvironment
         /// </summary>
         /// <returns>The environment variables as IDictionary&lt;string, string&gt; </returns>
```

Comment:
Is that part of this PR?  I don't recall doing anything with Environment variables here.